sha512_mb/sha512_mb_x2_avx.asm

```diff
@@ -337,10 +337,11 @@ Lrounds_16_xx:
 	vmovdqa	[STATE+7*SHA512_DIGEST_ROW_SIZE],h
 
 	; update input pointers
-	add	inp0, IDX
-	mov	[STATE + _data_ptr_sha512 + 0*PTR_SZ], inp0
-	add	inp1, IDX
-	mov [STATE + _data_ptr_sha512 + 1*PTR_SZ], inp1
+	vmovq	xmm0, IDX
+	vpunpcklqdq xmm0, xmm0
+	lea	IDX, [STATE + _data_ptr_sha512]
+	vpaddq	xmm0, xmm0, [IDX]
+	vmovdqu32 [IDX], xmm0
```

Comment:
Missed this one too: vmovdqu32 -> vmovdqu

Reply:
Done. Sorry, silly mistake on my part 😅